react/features/recording/components/LiveStream/AbstractStreamKeyForm.js

```diff
@@ -0,0 +1,111 @@
+// @flow
+
+import { Component } from 'react';
+
+declare var interfaceConfig: Object;
+
+/**
+ * The live streaming help link to display. On web it comes from
+ * interfaceConfig, but we don't have that on mobile.
+ *
+ * FIXME: This is in props now to prepare for the Redux-based interfaceConfig
+ */
+const LIVE_STREAMING_HELP_LINK = 'https://jitsi.org/live';
+
+/**
+ * The props of the component.
+ */
+export type Props = {
+
+    /**
+     * Callback invoked when the entered stream key has changed.
+     */
+    onChange: Function,
+
+    /**
+     * Invoked to obtain translated strings.
+     */
+    t: Function,
+
+    /**
+     * The stream key value to display as having been entered so far.
+     */
+    value: string
+};
+
+/**
+ * The state of the component.
+ */
+type State = {
+
+    /**
+     * The value entered in the field.
+     *
+     * NOTE: This has been refactored due to an error in the console on web.
```

Comment:
I'm not sure of the need for this comment. I won't have context on what was refactored.